### PR TITLE
HTML Lists Blog Posts

### PR DESCRIPTION
**Pull request by iamanolive**
Written articles for the HTML topic "Lists" as specified. Solves issue #106

### CODE REVIEW

File: docs/html/lists/ordered-lists.md [hunk 1]
```diff
@@ -1,8 +1,165 @@
...
 sidebar_position: 1
 tags: [html, web-development, ordered-lists, lists]
 description: In this tutorial, you will learn about ordered lists in HTML. Ordered lists are used to display a list of items in a specific order, such as numerical or alphabetical order.
----
\ No newline at end of file
+---
+
+# Unlocking the Potential of Ordered Lists in HTML: A Beginner-Friendly Guide
+
+Hello, coding aficionados and web development enthusiasts! Today, we’re going to explore the structured and orderly world of ordered lists in HTML. Whether you're a novice coder or a seasoned developer, understanding ordered lists can significantly enhance your web pages. So grab your favorite snack, get comfortable, and let's delve into the organized charm of <ol>, <li>, and the many ways they can be utilized.
```
Comment (Ajay-Dhangar) [inline docs/html/lists/ordered-lists.md:12]: **bad practice:**

```md
<ol>, <li>
```

Good Practice:

```md
`<ol>`, `<li>`
```


So, if you use any markup/html tag for example then use like ``` `<tagName>` ``` to use like visible or not count like as tag.

**Review by Ajay-Dhangar — CHANGES_REQUESTED** [review]
Follow right md (markdown) format.

File: docs/html/lists/ordered-lists.md (hunk 1 above)
Comment (iamanolive) [reply docs/html/lists/ordered-lists.md]: thank you so much for pointing that out @Ajay-Dhangar i'd added `` at most places, but missed a few, sorry. anything else i need to change?

**Review by Ajay-Dhangar — APPROVED** [review]
Done

**Pull request merged**